src/Raven.Server/Documents/AttachmentsStorageHelper.cs

```diff
@@ -42,6 +43,15 @@ public static async Task<string> CopyStreamToFileAndCalculateHash(JsonOperationC
             }
         }
 
+        public static string CalculateHash(ReadOnlySpan<byte> mem)
```

Comment:
This is basically how I would expect us to use `Blake2` in general

Reply:
done